dfs/snapshot.go

```diff
@@ -185,10 +185,19 @@ func (dfs *DistributedFilesystem) Rollback(snapshotID string, forceRestart bool)
 		return err
 	}
 
-	// rollback the dfs
-	glog.V(0).Infof("Performing rollback for %s (%s) using %s", tenant.Name, tenant.ID, snapshotID)
-	if err := snapshotVolume.Rollback(snapshotID); err != nil {
-		glog.Errorf("Error while trying to roll back to %s: %s", snapshotID, err)
+	err = func() error {
+		dfs.networkDriver.Stop()
```

Comment:
We should check the error code from Stop(). If we can't stop NFS, we should either skip the rollback, or at least report that error.